cdap-support-bundle/bin/observability.py

```diff
@@ -0,0 +1,539 @@
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": memory_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/appfabric/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-appfabric-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/metrics/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-metrics-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/messaging.service/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-messaging-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/log.saver/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-log-saver-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/runtime/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-runtime-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/status -H "{auth_token}"',
+            "fileName": 'kubectl-exec-system-service-status',
```

Comment:
The output looks like this:
```
Defaulted container "runtime" out of: runtime, systemmetricsexporter, storageinit (init)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0{"messaging.service":"OK","metrics.processor":"OK","appfabric":"OK","runtime":"OK","dataset.executor":"OK","metadata.service":"OK","metrics":"OK","log.saver":"OK"}
100   163  100   163    0     0   5258      0 --:--:-- --:--:-- --:--:--  5258
```
We just need the JSON, I think you can get rid of the other stuff if you ignore stderr

Reply:
done